assets/wire/utils.ts

```diff
@@ -108,10 +108,25 @@ function getBodyPicture(item: IArticle): IArticle | null {
     return pictures.length ? pictures[0] : null;
 }
 
-export function getPictureList(item: IArticle): Array<IArticle> {
+export function getPictureList(item: IArticle, options: {includeFeatured?: boolean, includeEditor?: boolean} = {}): IArticle[] {
```

Comment:
I think you can just destruct it with:
```ts
export function getPictureList(item: IArticle, {includeFeatured = false, includeEditor = false} = {}): IArticle[] {
```
but thinking about it it might be also useful to export a function which would have the params set like
`getGalleryMedia` that would call `getPictureList(item, {includeFeatured: false, includeEditor: false});`
so we don't have to use the params in preview/detail and we make sure it's consistent